src/state/apis/zerion/zerionApi.ts

```diff
@@ -22,33 +21,39 @@ const options = {
   },
 }
 
-// https://developers.zerion.io
+export const _getRelatedAssetIds = async (assetId: AssetId): Promise<AssetId[]> => {
```

Comment:
Extracted for use outside of RTK

Reply:
as morning brained memoize this bad boy

Reply:
5e9c0f3761bf8512f7439811bf26c9b4415ccc40